packages/next/build/webpack/plugins/next-esm-plugin.ts

```diff
@@ -234,6 +251,9 @@ export default class NextEsmPlugin implements Plugin {
         ).apply(childCompiler)
       }
     }
+    console.log('IN NEXT ESM PLUGIN')
```

Comment:
Should we remove this?

Reply:
whoops, didn't mean to push any changes to that file. let me make sure nothing else snuck in...